### PR TITLE
create main services component home page

### PR DESCRIPTION
**Pull request by RandSohail**
- create the main service component 
- main services + image link => props

### CODE REVIEW

File: client/src/Components/mainServices/MainServicesCard.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { Card } from 'antd';
+import 'antd/dist/antd.css';
+import './mainServices.css';
```
Comment (LinaYahya) [inline client/src/Components/mainServices/MainServicesCard.tsx:5]: rename file to style.css

File: client/src/Components/mainServices/MainServicesCard.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { Card } from 'antd';
+import 'antd/dist/antd.css';
```
Comment (LinaYahya) [inline client/src/Components/mainServices/MainServicesCard.tsx:4]:
```suggestion
```
No need to import as we import antd style at App.tsx

**Review by devstm — APPROVED** [review]

File: client/src/Components/mainServices/MainServicesCard.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+interface serviceProps {
+  service: string,
+  imgLink: number,
+}
+
+function MainServicesCard({ service, imgLink }: serviceProps) {
+  return (
+    <Card.Grid className="main-services-card">
+      <img src={`../../../images/${imgLink + 1}.png`} alt="main service we provide" className="main-services-crd-image" />
```
Comment (LinaYahya) [inline client/src/Components/mainServices/MainServicesCard.tsx:15]:
```suggestion
      <img src={`/images/${imgLink + 1}.png`} alt="main service we provide" className="main-services-crd-image" />
```
anything inside public simply have a path of /etc
and we can change alt to service text

File: client/src/Components/mainServices/index.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-len */
```
Comment (LinaYahya) [inline client/src/Components/mainServices/index.tsx:1]: fix eslint rules

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by abdulrahman-2020 — APPROVED** [review]

**Pull request merged**